python/instrumentation/openinference-instrumentation-crewai/src/openinference/instrumentation/crewai/_wrappers.py

```diff
@@ -0,0 +1,256 @@
+        return response
+
+
+class _KickoffWrapper:
+    def __init__(self, tracer: trace_api.Tracer) -> None:
+        self._tracer = tracer
+
+    def __call__(
+        self,
+        wrapped: Callable[..., Any],
+        instance: Any,
+        args: Tuple[Any, ...],
+        kwargs: Mapping[str, Any],
+    ) -> Any:
+        span_name = f"{instance.__class__.__name__}.kickoff"
+        with self._tracer.start_as_current_span(span_name) as span:
+            try:
+                crew = instance
+                inputs = kwargs.get("inputs", None) or (args[0] if args else None)
+
+                span.set_attribute("crew_key", crew.key)
+                span.set_attribute("crew_id", str(crew.id))
+                span.set_attribute("crew_inputs", json.dumps(inputs) if inputs else None)
+                span.set_attribute(
+                    "crew_agents",
+                    json.dumps(
+                        [
+                            {
+                                "key": agent.key,
+                                "id": str(agent.id),
+                                "role": agent.role,
+                                "goal": agent.goal,
+                                "backstory": agent.backstory,
+                                "verbose?": agent.verbose,
+                                "max_iter": agent.max_iter,
+                                "max_rpm": agent.max_rpm,
+                                "i18n": agent.i18n.prompt_file,
+                                "delegation_enabled": agent.allow_delegation,
+                                "tools_names": [tool.name.casefold() for tool in agent.tools or []],
+                            }
+                            for agent in crew.agents
+                        ]
+                    ),
+                )
+                span.set_attribute(
+                    "crew_tasks",
+                    json.dumps(
+                        [
+                            {
+                                "id": str(task.id),
+                                "description": task.description,
+                                "expected_output": task.expected_output,
+                                "async_execution?": task.async_execution,
+                                "human_input?": task.human_input,
+                                "agent_role": task.agent.role if task.agent else "None",
+                                "agent_key": task.agent.key if task.agent else None,
+                                "context": [task.description for task in task.context]
+                                if task.context
+                                else None,
+                                "tools_names": [tool.name.casefold() for tool in task.tools or []],
+                            }
+                            for task in crew.tasks
+                        ]
+                    ),
+                )
+
+                response = wrapped(*args, **kwargs)
```

Comment:
could the `try` be here instead? fewer indents that way
```suggestion
            try:
                response = wrapped(*args, **kwargs)
```

Reply:
done!